client/source/js/modules/model/cost-coverage.html

```diff
@@ -57,9 +57,29 @@
       <button ng-if="graphs || ccGraph" class="btn" ng-click="exportAll()" save-graph-as="export-all">Export all data</button>
 
       <h1>{{ displayedProgram.name }}</h1>
-      <div class="section elastic" style="width: 470px">
-        <div class="chart-container">
-          <div line-scatter-chart variant="ccGraph" data="ccGraph.data" options="ccGraph.options" save-graph-as></div>
+      <div class="section elastic">
+        <div class="elastic_col __shrink" style="width: 470px">
+          <div class="chart-container">
+            <div line-scatter-chart variant="ccGraph" data="ccGraph.data" options="ccGraph.options" save-graph-as></div>
+          </div>
+        </div>
+        <div class="elastic_col">
+          <p>
+        Specify the maximum funding (for plotting only):
+        <input type="number" class="txbox __inline __xl" placeholder="e.g.{{defaultXAxisMaximum}}"  ng-model="$parent.xAxisMaximum" ng-change="updateCurves()">
+          </p>
+          <p>
+          <label>
+            <input type="radio" name="displaycost" ng-model="$parent.displayCost" value="1">
+            Display program cost data in current prices
+          </label><br>
+          <label>
+            <input type="radio" name="displaycost" ng-model="$parent.displayCost" value="2">
+            Display program cost data in <input class="txbox __inline __l" value="2014" placeholder="e.g. 2014" ng-model="$parent.displayYear" min="projectInfo.dataStart" max="projectInfo.dataEnd"> prices
```

Comment:
@Ashalynd you specified value here? It's not working, because it's overwritten by ng-model. Did you want to have 2014 as default value or what was the intention?